olm-catalog/serverless-operator/manifests/operator_v1alpha1_knativekafka_crd.yaml

```diff
@@ -1,115 +1,121 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: knativekafkas.operator.serverless.openshift.io
 spec:
   group: operator.serverless.openshift.io
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        description: KnativeKafka is the Schema for the knativekafkas API
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            type: object
+            description: 'KnativeKafkaSpec defines the desired state of the KnativeKafka (from the client).'
+            properties:
+              channel:
+                description: Allows configuration for KafkaChannel installation
+                properties:
+                  bootstrapServers:
+                    description: BootstrapServers is comma separated string of bootstrapservers
+                      that the KafkaChannels will use
+                    type: string
+                  enabled:
+                    description: Enabled defines if the KafkaChannel installation
+                      is enabled
+                    type: boolean
+                required:
+                - enabled
+                - bootstrapServers
+                type: object
+              source:
+                description: Allows configuration for KafkaSource installation
+                properties:
+                  enabled:
+                    description: Enabled defines if the KafkaSource installation is
+                      enabled
+                    type: boolean
+                required:
+                - enabled
+                type: object
+          status:
+            type: object
+              description: 'KnativeKafkaStatus defines the observed state of KnativeKafka (from the controller).'
```

Comment:
This also seems to be indented incorrectly.

Reply:
yep :-)